views/random-beers.hbs

```diff
@@ -0,0 +1,12 @@
+<h1>Have random Beer!</h1>
+
+    <div >
+      {{#each response}}
+  <img src="{{image_url}}" alt="Poster for {{title}}">
+        <p>{{this.name}}</p>
+        <p>{{description}}</p>
+        <p>{{this.tagline}}</p>
+        <p>{{this.food_pairing}}</p>
```

Comment:
Here I would recommend you to loop over the food_pairing array ( {{#each}} ) so that you have more control over displaying the data within that array. Keep up the good work!